packages/meetup-shared/src/meetupPullRequest.ts

```diff
@@ -1,21 +1,16 @@
-import { extractMeetupDateAndTime } from './meetupForm';
+import { format } from 'date-fns';
 import { type Meetup } from './meetupType';
 
 export function getMeetupPullRequestContent(
   meetup: Meetup,
   issueNumber: number,
 ) {
-  const meetupDateAndTime = extractMeetupDateAndTime(meetup);
-
   return `New meetup
 
 Date:
-${meetupDateAndTime.date}
-
-Time:
-${meetupDateAndTime.time}
+${format(meetup.date, 'MMM do, yyyy @ HH:mm')}
```

Comment:
Magiaa. Tätä vois vähän avata.

```suggestion
${format(meetup.date, DATE_FORMAT)}
```

Ja jonnekkin tuonne ylä kulmaan top-level:

```ts
const DATE_FORMAT = [
  // Month as Jan, Feb, ..., Dec
  'MMM',
  // Date as 1st, 2nd, ..., 31st
  'do', 
  // Year as 0044, 0001, 1900, 2017	
  'yyyy' 
  // ???
  ,'@' ,
  // Hours and minutes as 01:02
  'HH:mm'
].join(' ')
```

Reply:
Hyvä idis, `@`-merkki vain coolina välissä, ei vaikuta formaattiin 😁